src/node_javascript.cc

```diff
@@ -17,14 +17,15 @@ using v8::Object;
 using v8::String;
 
 Local<String> MainSource(Environment* env) {
-  return OneByteString(env->isolate(), node_native, sizeof(node_native) - 1);
+  return OneByteString(env->isolate(), internal_node_native,
+                        sizeof(internal_node_native) - 1);
```

Comment:
I'm not really sure why `js2c` doesn't generate this as `internal_process_node_native`, but it doesn't..